cmd/operator-sdk/add/crd.go

```diff
@@ -32,6 +32,13 @@ import (
 // newAddCRDCmd - add crd command
 func newAddCRDCmd() *cobra.Command {
 	crdCmd := &cobra.Command{
+		Deprecated: `Use 'operaot-sdk add api' command to create a new API definition for a custom resource.  
+
+'add api' will generate or update CRDs for a particular GVK based on values ` +
```

Comment:
We probably don't need to discuss the details of another command in this message:
```suggestion
```